### PR TITLE
Added an option to do odigos version --cli/--cluster to print only th…

### PR DESCRIPTION
**Pull request by yodigos**
…e version.

Removed the need of adding the TAG in the building commands.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,11 @@
+NAMESPACE := odigos-system
+CONFIGMAP_NAME := odigos-deployment
+
 ORG := keyval
+
+# Cluster Version
+TAG := $(shell kubectl get configmap $(CONFIGMAP_NAME) -n $(NAMESPACE) -o jsonpath='{.data.ODIGOS_VERSION}')
```
Comment (RonFed) [inline Makefile:7]: Also possible is `TAG ?= $(shell odigos version --short)`

Eden has a similar change in the Makefile at: https://github.com/odigos-io/odigos/pull/1250/files

Comment (yodigos) [reply Makefile:7]: You are right!
In this PR I also added a flag for cli version and cluster version.
So, I'll use it :)

File: cli/cmd/version.go [hunk 2]
```diff
@@ -13,26 +13,44 @@ import (
...
+	OdigosDate           string
 )
 
 // versionCmd represents the version command
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print odigos version.",
 	Run: func(cmd *cobra.Command, args []string) {
-		short_flag, _ := cmd.Flags().GetBool("short")
+		short_flag, _ := cmd.Flags().GetBool(cliFlag)
```
Comment (blumamir) [inline cli/cmd/version.go:33]: The conventions for this project are camelCase 
```suggestion
		shortFlag, _ := cmd.Flags().GetBool(cliFlag)
```

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,12 @@
-TAG ?= $(shell odigos version --short)
+NAMESPACE := odigos-system
+CONFIGMAP_NAME := odigos-deployment
+
+TAG ?= $(shell odigos version --cluster)
 ORG := keyval
+
+# Cluster Version
+#TAG ?= $(shell kubectl get configmap $(CONFIGMAP_NAME) -n $(NAMESPACE) -o jsonpath='{.data.ODIGOS_VERSION}')
```
Comment (blumamir) [inline Makefile:8]: Do we still need it here?

File: Makefile [hunk 4]
```diff
@@ -1,5 +1,12 @@
-TAG ?= $(shell odigos version --short)
+NAMESPACE := odigos-system
+CONFIGMAP_NAME := odigos-deployment
```
Comment (blumamir) [inline Makefile:2]: these can be removed, right?

File: cli/cmd/version.go [hunk 5]
```diff
@@ -13,30 +13,65 @@ import (
...
 	Short: "Print odigos version.",
 	Run: func(cmd *cobra.Command, args []string) {
-		short_flag, _ := cmd.Flags().GetBool("short")
+		cliFlag, _ := cmd.Flags().GetBool(cliFlag)
+		clusterFlag, _ := cmd.Flags().GetBool(clusterFlag)
 
-		if short_flag {
+		if cliFlag {
 			fmt.Printf("%s\n", OdigosVersion)
+		}
```
Comment (yodigos) [inline cli/cmd/version.go:38]: @blumamir I changed it so you can do multiple arguments (cli + cluster).. I'm not sure there is or will be a use case for it. But, usually CLIs allow multiple arguments. So even in the future if we will want to add

**Review by blumamir — APPROVED** [review]

**Pull request merged**